ui/app/AppLayouts/Chat/controls/community/TokensListDropdownContent.qml

```diff
@@ -0,0 +1,113 @@
+import QtQuick 2.13
+import QtQuick.Layouts 1.14
+import QtQuick.Controls 2.13
+import QtGraphicalEffects 1.13
+
+import StatusQ.Core 0.1
+import StatusQ.Core.Theme 0.1
+import StatusQ.Controls 0.1
+import StatusQ.Components 0.1
+
+
+ListView {
+    id: root
+
+    property var headerModel
+
+    signal clickHeaderItem(int index)
+    signal clickItem(string name, string shortName, url imageSource)
```

Comment:
Suggest using passive form for those signals, e.g `headerItemClicked` or `itemClicked`